content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,211 @@
+* **DV** - Domain Validated - a certificate issuance process that involves the certificate authority validating the requester owns the domain. No validation of the business entity is performed.
+* **TLS** - Transport Layer Security
+* **DNS TXT record** - a "text" record in the Domain Name System (DNS).  TXT records usually take form of name=value.
+
+### Technical Requirements
+
+The following technical requirements are stated using [RFC-2119](https://datatracker.ietf.org/doc/html/rfc2119) terminology. These requirements focus on the security best practices that Cerner expects application developers to adhere to in order for Cerner to provide the best description of your application to end users:
+
+* Web applications MUST support the HTTPS 1.1 protocol.
+* Application SHOULD support TLS.  (Please refer to the [Cerner CODE Program's TLS guidance document](http://fhir.cerner.com/authorization/tls-guidance/) for further detailed practices related to TLS.)
+* Application SHOULD use (only) port 443 for TLS.
+* Application SHOULD support and prefer TLS 1.2 or higher.
+* Application SHOULD have a valid, non-expired TLS certificate issued from a [trusted authority](http://fhir.cerner.com/authorization/tls-guidance/).
+* Application SHOULD use an OV or EV certificate, if possible, instead of a DV certificate.
+* Application's web site SHOULD host a DNS TXT record containing the Cerner-issued client identifier for the application (also referred to as a "system account ID" in CernerCentral or a "client identifier" in the [OAuth 2 specification](https://datatracker.ietf.org/doc/html/rfc6749#page-15) ).
+* Application's name SHOULD ideally align with the name of the website hosting it and/or the legal entity operating it.
+* Mobile/Native applications SHOULD (as suggested by [RFC-8252](https://datatracker.ietf.org/doc/html/rfc8252) ) use [claimed "https" Scheme URI redirection](https://datatracker.ietf.org/doc/html/rfc8252#section-7.2) instead of private-use URI scheme redirection.
+
+
+Adhering to all the above requirements ensures that Cerner can convey the most meaningful, descriptive information about your application to end users. Failure to follow recommended (SHOULD) requirements will result in users seeing less information about your application. When Cerner is unable to determine information about your application because one or more of the requirements above have not been implemented, Cerner will display "unknown" when describing your application to users.  See the "Information Displayed to Users" section below for more details.
+
+If you'd like to better understand why Cerner has these technical requirements, please see the "*Information Displayed to Users*" and "*FAQ*" sections below.
+
+NOTE:  Please consider the [Cerner CODE Validation Program](https://code.cerner.com/apiaccess) if you would like Cerner to perform a deeper security assessment of your application.
```

Comment:
I think it's just called the "code program"

Maybe this is a better URL? https://code.cerner.com/submit

Reply:
@jfrhong - FWIW - I prefer the original link because it's the only place I could find where it speaks to "Value-Added Services & Fees" which is what a developer must pursue to become "Cerner Validated", right?

Reply:
Reformatted link but left it linking to https://code.cerner.com/apiaccess